content/es/community/static/cncf-code-of-conduct.md

```diff
@@ -0,0 +1,29 @@
+Código de Conducta para la Comunidad de la CNCF v1.0
+----------------------------------------------------
+
+### Código de Conducta para los Colaboradores
+
+Como colaboradores y mantenedores de este proyecto, y en el interés de promover una comunidad abierta y cordial, nos comprometemos a respetar a todas las personas que colaboren denunciando problemas, publicando solicitudes de función, actualizando documentación, presentando pedidos de validación o parches, entre otras actividades.
+
+Estamos comprometidos con hacer de la participación de todos los involucrados en este proyecto una experiencia libre de hostigamiento de cualquier índole, sin importar su nivel de experiencia, género, identidad y expresión de género, orientación sexual, discapacidad, aspecto personal, tamaño corporal, raza, grupo étnico, edad, religión ni su nacionalidad.
+
+Algunos ejemplos de conducta inaceptable imputable a los participantes pueden ser:
+
+-	El uso de palabras o imágenes sesgadas de contenido sexual.
+-	Ataques personales
+-	Comentarios provocadores, insultantes o despectivos
+-	Hostigamiento público o privado
+-	Publicar información ajena privada, como ser domicilios particulares o electrónicos, sin contar con permiso explícito para hacerlo
+-	Cualquier otra conducta reñida con la ética o el profesionalismo.
+
+Los mantenedores de proyectos tienen el derecho y la responsabilidad de eliminar, editar o rechazar los comentarios, consignas, códigos, correcciones colaborativas (del tipo “wiki edits”), ediciones y demás contribuciones que no estuvieren en consonancia con los principios establecidos en el presente Código de Conducta. Al adoptar este Código de Conducta, los mantenedores de proyectos se comprometen a aplicar estos principios de manera justa y uniforme a todos los aspectos que hacen a la gestión de este proyecto. Los mantenedores de proyectos que no cumplan ni hagan cumplir este Código de Conducta podrían quedar desafectados en forma permanente del equipo encargado del proyecto.
+
+Este Código de Conducta se aplica tanto dentro de los espacios relacionados con el proyecto como en espacios públicos y siempre que un individuo esté representando al proyecto o su comunidad.
+
+Los casos de comportamiento abusivo, acosador o de cualquier otro modo inaceptable podrán ser denunciados poniéndose en contacto con [Comité del Código de Conducta de Kubernetes](https://git.k8s.io/community/committee-code-of-conduct) al email <conduct@kubernetes.io>. Para otros proyectos, comuníquese con un mantenedor de proyectos de CNCF o con nuestra mediadora, Mishi Choudhary <mishi@linux.com>.
```

Comment:
Should this be: "poniéndose en contacto con el Comité" (missing "el")

Reply:
wow, that's the original one that can only be a exact copy (as stated in the README).

I just opened a PR to update the original: https://github.com/cncf/foundation/pull/34, the we will be able to update the website copy.

Reply:
Fixed in https://github.com/cncf/foundation/commit/e2a956a681f089ed9eddb9ba2eadc51f60658596 and updated in 21c69932e27b921cb657a09d7e7ff17e2a8938f0